src/main/java/net/sf/jabref/gui/help/NewVersionDialog.java

```diff
@@ -0,0 +1,105 @@
+/*  Copyright (C) 2016 JabRef contributors.
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+package net.sf.jabref.gui.help;
+
+import java.awt.Cursor;
+import java.awt.GridBagConstraints;
+import java.awt.GridBagLayout;
+import java.awt.Insets;
+import java.awt.event.MouseAdapter;
+import java.awt.event.MouseEvent;
+
+import javax.swing.JButton;
+import javax.swing.JDialog;
+import javax.swing.JFrame;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+
+import net.sf.jabref.gui.actions.version.IgnoreUpdateAction;
+import net.sf.jabref.gui.actions.version.RemindMeLaterAction;
+import net.sf.jabref.gui.actions.version.VisitDownloadPageAction;
+import net.sf.jabref.gui.desktop.JabRefDesktop;
+import net.sf.jabref.logic.l10n.Localization;
+import net.sf.jabref.logic.util.version.Version;
+
+public class NewVersionDialog extends JDialog {
+
+    public NewVersionDialog(JFrame frame, Version currentVersion, Version latestVersion, Version toBeIgnored) {
+        super(frame);
+        setTitle(Localization.lang("New version available"));
+
+        JLabel lblTitle = new JLabel(Localization.lang("A new version of JabRef has been released and can be updated now"));
+        JLabel lblCurrentVersion = new JLabel(Localization.lang("Current version") + ": " + currentVersion.getFullVersion());
```

Comment:
Maybe `Installed version` instead of current version (which may be mistaken for "current" = "latest")